Tools/clinic/clinic.py

```diff
@@ -2928,7 +2915,7 @@ def parse_arg(self, argname, displayname):
                 """.format(argname=argname, paramname=self.parser_name, cast=cast)
         return None
 
-    def set_template_dict(self, template_dict: dict[str, str]):
+    def set_template_dict(self, template_dict: TemplateDict):
```

Comment:
```suggestion
    def set_template_dict(self, template_dict: TemplateDict) -> None:
```
Should it be or just leave it as it is?